qcodes_contrib_drivers/drivers/Tektronix/Keithley_2000_Scan.py

```diff
@@ -0,0 +1,66 @@
+from functools import partial
+from typing import TYPE_CHECKING
+
+from qcodes.instrument import InstrumentChannel
```

Comment:
It's better to import from the full path: from qcodes.instrument.channel import InstrumentChannel